src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -757,6 +757,7 @@ export default class Pilot {
      * @return {array} [success of operation, readback]
      */
     initiateHoldingPattern(fixName, holdParameters) {
+        const radial = heading_to_string(holdParameters.inboundHeading + Math.PI);
```

Comment:
Can we call this `radialText` or something similar? I had to look a few times since I assumed this was simply the radial, when actually we're taking the radial value and making a string out of it. I'd name it `radialString` but then @n8rzz would slap me for using types in names :joy:

Reply:
☝️ maybe not slap, just say I was _'disappointed in you'_ 😉

Reply:
Everybody knows that's worse!